tmt/base.py

```diff
@@ -1339,7 +1339,7 @@ class Plan(
     _original_plan: Optional['Plan'] = None
     _remote_plan_fmf_id: Optional[FmfId] = None
 
-    _extra_L2_keys = [
+    extra_l2_keys = [
```

Comment:
Why is the underscore removed here? The list is not to be consumed by users outside of the `Plan` class.

Reply:
my bad, I broke it while doing one of the many rebases.